.storybook/OrangeTheme.js

```diff
@@ -35,8 +33,8 @@ export default create({
   inputTextColor: 'black',
   inputBorderRadius: 4,
 
-  brandTitle: 'Orange Storybook',
-  brandUrl: 'https://www.orange.com',
-  // TODO: change this URL
-  brandImage: 'https://www.orange.com/themes/theme_boosted/Master_Logo_RGB.png',
-});
\ No newline at end of file
+  // Brand
+  brandTitle: 'Orange Design',
+  brandUrl: 'https://design.orange.com/',
+  brandImage: 'https://design.orange.com/wp-content/themes/orangexdlab/assets/images/orange_logo.svg',
```

Comment:
Hell no, please use Boosted's dist logo instead. It's really, really better : 
1. Not another domain to get something from,
2. Responsive (includes the small logo) and optimized: see [Boosted v5 Design guidelines](https://v5-dev--boosted.netlify.app/docs/5.0/guidelines/logos/#responsive-svg).

Maybe send Boosted's logo to XD Labs x)

Reply:
Yep. It was just a test asked by the designers.

Reply:
Fixed with https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/565/commits/eed9b92ea8e67fa98122558b1d26f1e580c2d087